README.md

```diff
@@ -213,33 +214,26 @@ or
 bundle exec cap prod db_seed # for the prod servers
 ```
 
-### Populate the catalog
+### Populate the catalog for a single storage root
 
 In console, start by finding the storage root.
 
 ```ruby
 msr = MoabStorageRoot.find_by!(name: name)
-Audit::MoabToCatalog.seed_catalog_for_dir(msr.storage_location)
-```
-
-Or for all roots:
-```ruby
-MoabStorageRoot.find_each { |msr| Audit::MoabToCatalog.seed_catalog_for_dir(msr.storage_location) }
```

Comment:
at first i was going to fix this to be `CatalogUtils.populate_catalog_for_all_storage_roots`, since that method iterates over all the storage roots, and there aren't enough storage roots to make a `find_each` (which retrieves query results in batches) necessary for performance reasons.  then i realized we already give instructions above about using that method to populate the catalog for all storage roots, so this felt redundant.